avocado/core/html.py

```diff
@@ -115,7 +115,7 @@ def _get_sysinfo(self, sysinfo_file):
         return sysinfo_contents
 
     def hostname(self):
-        return self._get_sysinfo('hostname')
+        return self._get_sysinfo('hostname').strip()
```

Comment:
Btw how about putting the `stript` directly to the `_get_sysinfo`. Aren't all of the sysinfo alike?